src/main/resources/data-repository-openapi.yaml

```diff
@@ -77,95 +80,161 @@ paths:
         500:
           description: service is broken
 
-  ## Access Interface
+  ## Repository Interface
 
-  '/api/access/v1/dataset':
-    get:
+  '/api/repository/v1/datasets':
+    get:  ## enumerate
       description: |
         Returns a list of all of the datasets the caller has access to
       operationId: enumerateDatasets
       tags:
-        - access
-      parameters:
-        - in: query
-          name: containing
-          type: string
-          description: return only datasets with name or description containing the string
+      - repository
       responses:
         200:
           description: List of datasets
           schema:
             type: array
             items:
-              $ref: '#/definitions/DatasetModel'
+              $ref: '#/definitions/DatasetSummary'
 
-  ## Search Interface
-  '/api/search/v1/dataset':
-    get:
-      description: |
-        Given a search query in the body, returns a list of matching dataset ids
-      operationId: datasetSearch
+    post:
+      description: Create a new dataset
+      operationId: createDataset
+      tags:
+      - repository
+      parameters:
+      - in: body
+        name: dataset
+        description: Dataset to create
+        schema:
+          $ref: '#/definitions/DatasetRequest'
+      responses:
+        201:
```

Comment:
this should actually be a 202: Accepted which will have a location url in the header response that can be used to check job status. the actual DatasetSummary object will be returned from a get with the appropriate id. And the description should say that this kicks off a create dataset job.